### PR TITLE
Use EphemeralStorage class from libmozevent to store builds and pushes related to test selection

### LINKED ISSUE #327: Store diff_to_push and task_group_to_build in durable storage
We still want to keep them in memory for performance reasons (when acting on pulse messages, we want to be extra quick).
So we could:
1) Store them in memory and in durable storage;
2) At startup, read them from durable storage to memory.

### PR DESCRIPTION
**Pull request by marco-c**
Fixes #327

Requires a new version of libmozevent, which I will release tomorrow.

### CODE REVIEW

**Comment by marco-c** [comment]
I've rebased on top of the libmozevent 1.1.0 update, it should be ready now.

File: events/code_review_events/bugbug_utils.py [hunk 1]
```diff
@@ -63,23 +70,34 @@ def __init__(self):
...
 
         # Store the push revision and build, so we can use it after bugbug
         # selects tests to add.
-        self.diff_to_push[int(build.diff_id)] = {
-            "revision": extras["revision"],
-            "treeherder_url": extras["treeherder_url"],
-            "build": build,
-        }
+        await self.diff_to_push.set(
+            int(build.diff_id),
```
Comment (marco-c) [inline events/code_review_events/bugbug_utils.py]: I've just realized I might need to use strings as keys everywhere, let me test...

Comment (marco-c) [reply events/code_review_events/bugbug_utils.py]: Yes, I needed to do that, and I've done it now.

File: events/code_review_events/bugbug_utils.py [hunk 2]
```diff
@@ -63,23 +69,34 @@ def __init__(self):
         self.hooks_service = taskcluster_config.get_service("hooks")
         self.queue_service = taskcluster_config.get_service("queue")
 
+    async def setup(self):
+        self.task_group_to_build = await EphemeralStorage.create(
```
Comment (La0) [inline events/code_review_events/bugbug_utils.py]: :+1:

File: events/code_review_events/bugbug_utils.py [hunk 3]
```diff
@@ -255,7 +272,11 @@ def add_new_jobs(self, revision, selected_tasks):
             return
 
         # If this diff does not belong to a revision we pushed to try, return.
-        if diff_id not in self.diff_to_push:
+        try:
+            push = self.diff_to_push.get(diff_id)
```
Comment (La0) [inline events/code_review_events/bugbug_utils.py]: NO await on get ?

Comment (marco-c) [reply events/code_review_events/bugbug_utils.py]: `get` only uses the memory cache for perf reasons, without accessing Redis

**Pull request merged**